react/Summary.tsx

```diff
@@ -11,7 +11,7 @@ interface Props {
 
 const Summary: FC<Props> = ({ classes }) => {
   const {
-    orderForm: { totalizers, value, paymentData },
+    orderForm: { totalizers, value, paymentData},
```

Comment:
Fix this lint issue.
```suggestion
    orderForm: { totalizers, value, paymentData },
```